### PR TITLE
Update Stitches Info

### PR DESCRIPTION
**Pull request by peduarte**
Hey Andrei 👋 

What a great idea you had with this repo. It's very well thought out and executed. Good work!

Stitches has since gone through a few iterations, so this PR is updating some things that have changed. 

Thanks 🔥 

PS: I've also updated the demo app, PR here https://github.com/andreipfeiffer/css-in-js/pull/18

### CODE REVIEW

**Comment by peduarte** [comment]
I've not updated this table: 
image

How are you getting this data?

File: README.md [hunk 1]
```diff
@@ -1626,7 +1623,7 @@ Some upsides:
 - successfully bundles only used styles, doesn't bundle all classes defined in `tailwind.config`
    - exception: keyframe animations (spin, ping, etc)
    - beware of purging
-   
+
```
Comment (peduarte) [inline README.md:1626]: prettier

File: README.md [hunk 2]
```diff
@@ -819,7 +819,7 @@ Version: __`11.1`__ | Maintained by [Mitchell Hamilton](https://twitter.com/mitc
 
 - __Styles output__
   - ❌ `.css` file extraction
-  - ✅ `<style>` tag injection  
+  - ✅ `<style>` tag injection
```
Comment (peduarte) [inline README.md:822]: prettier

File: README.md [hunk 3]
```diff
@@ -468,15 +468,15 @@ Because we're shipping less CSS bytes, and they are inlined inside the `.html` f
...
+📬 **Full cache**
+When the user's cache is full, the additional `.js` files won't require fetching, as they are already cached.
 However, if the page is **SSRed**, the inlined critical CSS rendered in the `<style>` tag of the document will be downloaded again, unless we deal with static HTML that can be cached as well, or we deal with HTML caching on our infrastructure.
 
 But, by default, we will ship extra bytes on every page HTTP request, regardless if it's cached or not.
 
 <br />
 
-💡 **Key points**  
+💡 **Key points**
```
Comment (peduarte) [inline README.md:479]: prettier

File: README.md [hunk 4]
```diff
@@ -468,15 +468,15 @@ Because we're shipping less CSS bytes, and they are inlined inside the `.html` f
    - the styles required for the page, bundled in `.js` files along with the components, during [hydration](https://developers.google.com/web/updates/2019/02/rendering-on-the-web#rehydration-issues) (this includes all the critical CSS already shipped inside the `<style>` tag + others);
 - all these files are required to be fetched, parsed and executed to get a **fully interactive** page;
 
-📬 **Full cache**  
-When the user's cache is full, the additional `.js` files won't require fetching, as they are already cached.  
+📬 **Full cache**
```
Comment (peduarte) [inline README.md:471]: prettier

File: README.md [hunk 5]
```diff
@@ -458,7 +458,7 @@ This solution appears to be better suited when:
 
 During **SSR**, styles will be added as `<style>` tag(s) in the `<head>` of the page. Keep in mind that these usually do NOT include all styles needed for the page, because most libraries perform [Critical CSS extraction](#-critical-css-extraction), so these `styles` should be usually smaller than the entire `.css` static file discussed previously.
 
-📭 **Empty cache**  
+📭 **Empty cache**
```
Comment (peduarte) [inline README.md:461]: prettier

File: README.md [hunk 6]
```diff
@@ -441,12 +441,12 @@ If the user has an empty cache, the following needs to happen, **negatively** im
...
 It's true that you can fetch in **parallel** other `<head>` resources (additional `.css` or `.js` files), but this is generally a bad practice;
 
-📬 **Full cache**  
+📬 **Full cache**
 However, on subsequent visits, the entire `.css` resource would be cached, so **FCP** and **LCP** would be positively impacted.
 
 <br />
 
-💡 **Key points**  
+💡 **Key points**
```
Comment (peduarte) [inline README.md:449]: prettier

File: README.md [hunk 7]
```diff
@@ -427,11 +427,11 @@ Also, there are 2 different scenarios we need to consider:
 
 <br />
 
-#### 1. `.css` file extraction  
+#### 1. `.css` file extraction
 
 Solutions that generate `.css` static files, which you normally would include as `<link>` tag(s) in the `<head>` of your page, are basically [rendering-blocking resources](https://developers.google.com/web/fundamentals/performance/critical-rendering-path/render-blocking-css). This highly affects **FCP**, **LCP** and any other metric that follows.
 
-📭 **Empty cache**  
+📭 **Empty cache**
```
Comment (peduarte) [inline README.md:434]: prettier

File: README.md [hunk 8]
```diff
@@ -427,11 +427,11 @@ Also, there are 2 different scenarios we need to consider:
 
 <br />
 
-#### 1. `.css` file extraction  
+#### 1. `.css` file extraction
```
Comment (peduarte) [inline README.md:430]: prettier

File: README.md [hunk 9]
```diff
@@ -246,7 +246,7 @@ We **haven't tested out this feature**, so we're only taking notes which librari
 #### 9. `.css` (Static CSS extraction)
 
 Defined styles are extracted as static `.css` files:
- 
+
```
Comment (peduarte) [inline README.md:249]: prettier

File: README.md [hunk 10]
```diff
@@ -325,7 +325,7 @@ Allows passing styles using a special `css` prop, similar how we would define in
 
 #### 15. Framework agnostic
 
-Allows usage without, or with any framework. Some libraries are built specifically for React only.  
+Allows usage without, or with any framework. Some libraries are built specifically for React only.
```
Comment (peduarte) [inline README.md:328]: prettier

File: README.md [hunk 11]
```diff
@@ -110,13 +110,13 @@ This analysis is intended to be **objective** and **unopinionated**:
...
 
-👎 **What you WON'T FIND here?**  
+👎 **What you WON'T FIND here?**
 - which solution is _"the best"_, as I'll not add any grading, which would also be highly subjective;
 - which solution is _"the fastest"_, as I'm not concearned about rendering performance metrics (you can checkout [Necholas's benchmarks](https://necolas.github.io/react-native-web/benchmarks/) for this);
 
 <br />
 
-👍 **What you WILL FIND here?**  
+👍 **What you WILL FIND here?**
```
Comment (peduarte) [inline README.md:119]: prettier

File: README.md [hunk 12]
```diff
@@ -110,13 +110,13 @@ This analysis is intended to be **objective** and **unopinionated**:
 
 <br />
 
-👎 **What you WON'T FIND here?**  
+👎 **What you WON'T FIND here?**
```
Comment (peduarte) [inline README.md:113]: prettier

File: README.md [hunk 13]
```diff
@@ -64,12 +64,12 @@ The **CSS language** and **CSS Modules** have some limitations, especially if we
...
 
-3. **Styles usage is disconnected from their definition**  
+3. **Styles usage is disconnected from their definition**
   We get no IntelliSense with CSS Modules, of what CSS classes are defined in the `component.css` file, making **copy-paste** a required tool, lowering the DX. It also makes __refactoring very cumbersome__, because of the lack of safety.
 
-4. **Using type-safe design tokens in CSS is non-trivial**  
+4. **Using type-safe design tokens in CSS is non-trivial**
   Any [design tokens](https://spectrum.adobe.com/page/design-tokens/) defined in JS/TS (to benefit from type-safety) cannot be directly used in CSS.
-  
+
```
Comment (peduarte) [inline README.md:72]: prettier

File: README.md [hunk 14]
```diff
@@ -64,12 +64,12 @@ The **CSS language** and **CSS Modules** have some limitations, especially if we
     }
     ```
 
-3. **Styles usage is disconnected from their definition**  
+3. **Styles usage is disconnected from their definition**
```
Comment (peduarte) [inline README.md:67]: prettier

File: README.md [hunk 15]
```diff
@@ -45,12 +45,12 @@ Last important aspect is type-safety with full **TypeScript** support.
...
 
-1. **Styles cannot be co-located with components**  
+1. **Styles cannot be co-located with components**
   This can be frustrating when authoring many small components, but it's not a deal breaker. However, the experience of moving back-and-forth between the `component.js` file and the `component.css` file, searching for a given class name, and not being able to easily _"go to style definition"_, is an important productivity drawback.
 
-2. **Styling pseudos and media queries requires selector duplication**  
+2. **Styling pseudos and media queries requires selector duplication**
   Another frustrating fact is the need to duplicate our CSS classes when defining __pseudo classes and elements__, or __media queries__. We can overcome these limitations using a CSS preprocessor like __SASS, LESS or Stylus__, that supports the `&` parent selector, enabling __contextual styling__.
-  
+
```
Comment (peduarte) [inline README.md:53]: prettier

File: README.md [hunk 16]
```diff
@@ -45,12 +45,12 @@ Last important aspect is type-safety with full **TypeScript** support.
 
 The **CSS language** and **CSS Modules** have some limitations, especially if we want to have type-safe code. Some of these limitations have alterative solutions, others are just being _annoying_ or _less than ideal_:
 
-1. **Styles cannot be co-located with components**  
+1. **Styles cannot be co-located with components**
   This can be frustrating when authoring many small components, but it's not a deal breaker. However, the experience of moving back-and-forth between the `component.js` file and the `component.css` file, searching for a given class name, and not being able to easily _"go to style definition"_, is an important productivity drawback.
 
-2. **Styling pseudos and media queries requires selector duplication**  
+2. **Styling pseudos and media queries requires selector duplication**
```
Comment (peduarte) [inline README.md:51]: prettier

File: README.md [hunk 17]
```diff
@@ -2,8 +2,8 @@
 
 This document contains an in-depth analysis of all the current **CSS-in-JS** solutions, that support **Server Side Rendering** and **TypeScript**.
 
-The baseline reference we'll use for comparison is a **CSS Modules** approach.  
-We're using **Next.js** as a SSR framework for building resources.  
+The baseline reference we'll use for comparison is a **CSS Modules** approach.
```
Comment (peduarte) [inline README.md:5]: prettier

**Comment by andreipfeiffer** [comment]
Thanks you very much for this PR and the one that updated the example 🤘

I have updated the Stitches review:
https://github.com/andreipfeiffer/css-in-js/blob/main/README.md#stitches

**Pull request closed without merging**